spec/controllers/categories_controller_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe CategoriesController do
+  describe 'GET #index' do
+  	before(:each) { @category = create(:category) }
```

Comment:
@category = create(:cat...) przenieś do pierwszego it'a bo tam tylko sprawdzasz kategorie więc nie ma potrzeby generować tego dla każdego it'a